jekyll/_cci2/workflows.adoc

```diff
@@ -188,16 +147,36 @@ workflows:
             - acceptance_test_4
 ----
 
-In this example, as soon as the `build` job finishes successfully, all four acceptance test jobs start. The `deploy` job must wait for all four acceptance test jobs to complete successfully before it starts.
+In this example, as soon as the `build` job finishes successfully, all four acceptance test jobs start. The `deploy` job waits for all four acceptance test jobs to succeed before it starts.
 
 See the link:https://github.com/CircleCI-Public/circleci-demo-workflows/tree/fan-in-fan-out[Sample Fan-in/Fan-out Workflow config] for a full example.
 
 [#holding-a-workflow-for-a-manual-approval]
 == Hold a workflow for a manual approval
 
-Configure a workflow to wait for manual approval before continuing using an `approval` job. Anyone who has push access to the repository can click the *Approve* button in the CircleCI web app to continue the workflow.
+Use an `approval` job to configure a workflow to wait for manual approval before continuing. Anyone who has push access to the repository can approve the job to continue the workflow. Approve either by using the *Approve* button in the CircleCI web app, or via the API.
+
+Some things to keep in mind when using manual approval in a workflow:
+
+* `approval` is a special job type that is configured when listing jobs under the `workflow` key.
+* The approval job name must be a unique and not used by any other job in your configuration. For example, your configured jobs, such as `build` or `test1` in the example below, would not be set as `type: approval`.
```

Comment:
Remove the `a` in "The approval job name must be **a** unique and not used by..."

Reply:
```
For example, your configured jobs, such as `build` or `test1` in the example below, would not be set as `type: approval`.
```

I don't understand how this relates to approval job names being unique. The jobs `build` or `test1` in the example could  have `type: approval` and the workflow would run fine.